ow_power_system/src/power_system_node.cpp

```diff
@@ -24,6 +24,10 @@ const string FAULT_NAME_THERMAL = "thermal_power_failure";
 //
 static constexpr int TEMPERATURE_INDEX = 1;
 
+// GLOBAL VARS (should implementation be changed to avoid this?)
+bool one_fault_injected = false;
+string one_fault_name = "";
```

Comment:
I'll suggest a clearer name for this one, `SelectedFaultName`.  Also add `static` to make this a file variable.